SplitScreenScanner/Classes/SplitScannerCoordinator.swift

```diff
@@ -11,6 +11,18 @@ import AVFoundation
 public protocol SplitScannerCoordinatorDelegate: class {
     func didScanBarcode(_ splitScannerCoordinator: SplitScannerCoordinator, barcode: String) -> ScanResult
     func didPressDoneButton(_ splitScannerCoordinator: SplitScannerCoordinator)
+
+    // Optional
+    func shouldDismiss(after scanResult: ScanResult) -> Bool
+    func didTriggerDismissalWithScan(_ splitScannerCoordinator: SplitScannerCoordinator)
```

Comment:
The user shouldn't need to set this as SplitScreenScanner should have functionality for closing out itself.